### PR TITLE
Problem #65: Apply the new jargon

### PR DESCRIPTION
**Pull request by kegor**
Initial iteration for #65.

Working on fixing tests right now.

### CODE REVIEW

**Comment by kegor** [comment]
For now, only tests that already mentioned in #67 are failed.

File: src/BlendingEditor/Website/ViewModels/BaseLayoutConditionsPage.json.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            this.ModalOpened = true;
+        }
+
+        public void Handle(Input.SaveChangesTrigger action)
+        {
+            foreach (var layoutCondition in this.LayoutConditions)
+            {
+                layoutCondition.HeadersString = layoutCondition.Headers.Select(x => x.Data).GetHeadersString();
+                var emptyHeaders = layoutCondition.Headers.Where(h => string.IsNullOrWhiteSpace(h.Name)).ToList();
+                foreach (var routeHeader in emptyHeaders)
```
Comment (PatrykSzwer) [inline src/BlendingEditor/Website/ViewModels/BaseLayoutConditionsPage.json.cs]: there are still some occurrences of `routeHeader`. Please fix here and in rest places.

File: src/BlendingProvider/Website/Api/BlendingHandlers.cs [hunk 2]
```diff
@@ -85,12 +85,12 @@ public void Register()
         /// <summary>
         /// Map Blending Rule
```
Comment (PatrykSzwer) [inline src/BlendingProvider/Website/Api/BlendingHandlers.cs]: There are still some comments that refers to `blending rules` which should be changed to are items. Please change them.

Comment (kegor) [reply src/BlendingProvider/Website/Api/BlendingHandlers.cs]: Done.

File: src/BlendingEditor/Website/ViewModels/BaseLayoutConditionsPage.json.cs (hunk 1 above)
Comment (kegor) [reply src/BlendingEditor/Website/ViewModels/BaseLayoutConditionsPage.json.cs]: Done.

**Comment by warpech** [comment]
Please merge current `master` to this branch. It should succeed now.

**Comment by warpech** [comment]
In https://github.com/Starcounter/Blending/pull/66/commits/6d648a2e93d67a88e99071f0e217133b9a2eaaca, I fixed few instances where there was some error in translation (the term"area item" was used for header rules of layout conditions). @PatrykSzwer could you please double check if there are no other mistranslations like that?

**Pull request merged**